### PR TITLE
Added header area and tag for inactive groups

### PR DESCRIPTION
**Pull request by oratiu**

### CODE REVIEW

File: packages/react-components/src/templates/GroupProfilePage.tsx [hunk 1]
```diff
@@ -12,10 +13,16 @@ const mainStyles = css({
...
+  active,
   ...props
 }) => (
   <article>
-    <GroupProfileHeader {...props} />
+    {!active && (
+      <Toast accent="info">
+        This group is inactive and might not have all content available.
+      </Toast>
+    )}
```
[the remark this answers is not in the document]
Comment (oratiu) [reply packages/react-components/src/templates/GroupProfilePage.tsx:24]: Thank you for catching that, idk how it slipped from me.

**Review by lctrt — APPROVED** [review]

File: packages/react-components/src/organisms/Toast.tsx [hunk 2]
```diff
@@ -72,21 +80,38 @@ const crossPlaceholderStyles = css(iconStyles, {
 const wrapStyles = css({
   display: 'flex',
   flexWrap: 'wrap',
+  flexFlow: 'row',
 });
 
+export const getIcon = (accent?: 'alert' | 'info'): EmotionJSX.Element => {
```
Comment (kmaid) [inline packages/react-components/src/organisms/Toast.tsx:86]: Wouldn't this be better expressed as a Record and set a default accent in the props?

File: packages/react-components/src/organisms/Toast.tsx [hunk 3]
```diff
@@ -72,21 +80,38 @@ const crossPlaceholderStyles = css(iconStyles, {
...
   children: ReactNode;
   onClose?: () => void;
+  accent?: 'alert' | 'info';
 }
-const Toast: React.FC<ToastProps> = ({ children, onClose }) => (
-  <section css={styles}>
+const Toast: React.FC<ToastProps> = ({
+  children,
+  onClose,
+  accent = 'alert',
```
Comment (kmaid) [inline packages/react-components/src/organisms/Toast.tsx:105]: This means your default case can't ever be reached no?

File: packages/react-components/src/icons/info-circle-yellow.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+/* istanbul ignore file */
+
+const infoCircleYellow = (
```
Comment (kmaid) [inline packages/react-components/src/icons/info-circle-yellow.tsx:3]: This should be added to the icons storybook. Surprised we don't already have this

File: apps/storybook/src/Icons.stories.tsx [hunk 5]
```diff
@@ -40,6 +40,7 @@ export const Groups = () => components.groupsIcon;
 export const HidePassword = () => components.hidePasswordIcon;
 export const Info = () => components.infoIcon;
 export const InfoCircle = () => components.infoCircleIcon;
+export const infoCircleYellow = () => components.infoCircleYellow;
```
Comment (ricardocarvalh0) [inline apps/storybook/src/Icons.stories.tsx:43]: Shouldn't the name start with upper case as the others?

**Review by ricardocarvalh0 — APPROVED** [review]

**Pull request merged**